### PR TITLE
publish developer resources

### PR DESCRIPTION
**Pull request by kne42**
# Description
Publish developer resources (e.g. governance model, licensing, contributor guides, etc.)

## Type of change

- [x] Documentation

# References

see https://github.com/napari/napari.github.io/pull/1#discussion_r377368975

### CODE REVIEW

**Comment by kne42** [comment]
@tlambert this gets rendered like in the following screenshot

Screen Shot 2020-02-11 at 4 22 16 PM

how do we make it so that it shows all of the different files instead of the section titles?

File: docs/source/developers.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Developer Resources
+===================
+
+Resources for the governance of the project
+and the onboarding of new contributors.
+
+.. toctree::
+   :maxdepth: 1
```
Comment (jni) [inline docs/source/developers.rst:8]: @kne42 maybe changing maxdepth to 0 here would work?

**Review by jni — APPROVED** [review]
Imma approve this for now so I'm not holding it up, looks pretty great so far. =)

**Comment by tlambert03** [comment]
> how do we make it so that it shows all of the different files instead of the section titles? 

Hi @kne42, sorry for the delay on this.  After taking a look, I think the best approach is probably not to change the toc directive, but rather change the md files themselves (they use headings a little loosely... and sphinx pays close attention to the exact number of `#` in the header).  

Basically, there should only be a single h1 `# Header` (and that will be what the page is called in the nav).  For instance, `CODE_OF_CONDUCT` has an h1 `# Endnotes` title at the bottom of the doc, so it gets rendered in the nav.  `GOVERNANCE.md` starts with the h1 `# Abstract`, when it should probably start with `# Governance`, and make abstract and h2, etc...
(I use [markdownlint](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md) in my IDE, which can be helpful to highlight all these types of things that can lead to unexpected results.)

Let me know if you'd like me to push some changes directly...

File: docs/Makefile [hunk 2]
```diff
@@ -28,6 +28,8 @@ clean:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	make buildapi
+	rm -rf source/developers
+	cp -R developers source/
```
Comment (tlambert03) [inline docs/Makefile:33]: we could actually just consider putting this in source.  The only reason I didn't with release notes is that it felt like they belonged higher.  But these really are docs, and could live there.

Comment (tlambert03) [reply docs/Makefile:33]: also, if we _do_ keep the `developers` docs folder where it is and just copy it when building docs, don't forget to add `docs/source/developers/` to `.gitignore`

**Comment by sofroniewn** [comment]
I'm bumping this to `0.2.13`, though it doesn't really affect our release schedule. I can sync with @kne42 tomorrow about next steps here. Would be really nice to get this in

**Comment by kne42** [comment]
@tlambert03 I put in the links manually but it's still expanding the code of conduct, the report handling manual, and the governance model documents:
Screen Shot 2020-03-13 at 11 52 29 AM

Any ideas on how to fix this?

**Comment by tlambert03** [comment]
> @tlambert03 I put in the links manually but it's still expanding the code of conduct, the report handling manual, and the governance model documents. Any ideas on how to fix this?

Hi @kne42, did you see this https://github.com/napari/napari/pull/967#issuecomment-590008419?

**Comment by kne42** [comment]
@tlambert03 No, I totally missed that! Will give it a try now...

**Comment by kne42** [comment]
Hi all, this is ready for review. Please take a quick look at the docs to make sure everything looks alright and then merge :)

**Review by sofroniewn — APPROVED** [review]
LGTM! Thanks!! I'll wait to see if @tlambert03 or @jni want to weigh in again and then merge.

**Comment by sofroniewn** [comment]
@kne42 @tlambert03 can you maybe look into why all the tests are failing - it seems to be specific to this PR as I have re-run them recently and seen other PRs be ok

**Comment by tlambert03** [comment]
it's all the `TypeError: ChannelABC() takes no arguments` as in #1054

**Comment by sofroniewn** [comment]
ah - maybe @kne42 can update from master with the version pin and then push again?

**Comment by sofroniewn** [comment]
Thanks for the fix! Will merge now :-)

**Pull request merged**